### PR TITLE
Feat/rebrandartcodix

### PR DESCRIPTION
**Pull request by chrisschwa**
changed artcodix corporate form to GmbH and changed the URL of the Link aswell because it passes to the old website instead of the cloud portal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This is a list of clouds that we test on a nightly basis against our `scs-compat
...
-| [CNDS](https://cnds.io/)                                                                                       | Public cloud for customers                        | [artcodix UG](https://artcodix.com/) | [![Compliance Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-artcodix-v3.yml?label=v3)](https://github.com/SovereignCloudStack/standards/actions/workflows/check-artcodix-v3.yml)  |                                 [HM](https://ohm.muc.cloud.cnds.io/)                                              |
+| [CNDS](https://cnds.io/)                                                                                       | Public cloud for customers                        | [artcodix GmbH](https://portal.cnds.io/) | [![Compliance Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-artcodix-v3.yml?label=v3)](https://github.com/SovereignCloudStack/standards/actions/workflows/check-artcodix-v3.yml)  |                                 [HM](https://ohm.muc.cloud.cnds.io/)                                              |
```
Comment (berendt) [inline README.md:16]: I would prefer to remove the artcodix GmbH link (like for all of the other companies) or to link the company website itself. CNDS is already linked to cnds.io.

Comment (fkr) [reply README.md:16]: Yes, I was about to suggest the same.
Imho the name should link to the site that displays the offering and that should be the only link - just like with all other entries in the table.

Comment (chrisschwa) [reply README.md:16]: i guess you are right. either remove it or change it back. ill commit an update

Comment (chrisschwa) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ This is a list of clouds that we test on a nightly basis against our `scs-compat
...
-| [CNDS](https://cnds.io/)                                                                                       | Public cloud for customers                        | [artcodix UG](https://artcodix.com/) | [![Compliance Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-artcodix-v3.yml?label=v3)](https://github.com/SovereignCloudStack/standards/actions/workflows/check-artcodix-v3.yml)  |                                 [HM](https://ohm.muc.cloud.cnds.io/)                                              |
+| [CNDS](https://cnds.io/)                                                                                       | Public cloud for customers                        | [artcodix GmbH](https://artcodix.com) | [![Compliance Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-artcodix-v3.yml?label=v3)](https://github.com/SovereignCloudStack/standards/actions/workflows/check-artcodix-v3.yml)  |                                 [HM](https://ohm.muc.cloud.cnds.io/)                                              |
```
Comment (fkr) [inline README.md:16]:
```suggestion
| [CNDS](https://cnds.io/)                                                                                       | Public cloud for customers                        | artcodix GmbH | [![Compliance Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-artcodix-v3.yml?label=v3)](https://github.com/SovereignCloudStack/standards/actions/workflows/check-artcodix-v3.yml)  |                                 [HM](https://ohm.muc.cloud.cnds.io/)                                              |
```

is what I meant :)

**Review by mbuechse — APPROVED** [review]
formal approve

**Pull request merged**